src/django_components/component.py

```diff
@@ -76,14 +75,15 @@
 
 # isort: on
 
-RENDERED_COMMENT_TEMPLATE = "<!-- _RENDERED {name} -->"
```

Comment:
Moved `RENDERED_COMMENT_TEMPLATE` inside `dependencies.py`, so all related code is in that file.